### PR TITLE
Translate 2015-11-11-ruby-2-3-0-preview1-released.md [vi]

### PR DESCRIPTION
**Pull request by joneslee85**

### CODE REVIEW

File: vi/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Chúng tôi hân hạnh công bố ấn bản Ruby 2.3.0-preview1.
+
+Ruby 2.3.0-preview1 là phiên bản thử nghiệm đầu tiên của Ruby 2.3.0.
+Giới thiệu cùng với phiên bản này bao gồm các cải tiến sau:
+
+[Frozen String Literal
+Pragma](https://bugs.ruby-lang.org/issues/11473) được giới thiệu. Trên
+Ruby 2.1, "str".freeze được tối ưu hoá để giảm số lượng đối tượng được
+tạo ra. Ruby 2.3 cho phép đóng băng tất cả string literal trong các
```
Comment (huydx) [inline vi/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md]: s/string literal/chuỗi kí tự/

**Comment by nguyenducgiang** [comment]
:+1:

**Comment by huynhquancam** [comment]
:+1:

**Comment by stomar** [comment]
Please squash commits with tiny fixes before merging, and also avoid non-ASCII characters in commit messages.

**Pull request merged**